specs/jsonschema-validation.md

```diff
@@ -328,10 +328,10 @@ an annotation.
 
 Implementations:
 
-- SHOULD provide validation for each format attribute defined in this
-  document;
-- MAY support format values not defined in this document, but such support MUST
-  be configurable and disabled by default;
+- SHOULD provide validation for each format attribute defined in this document
+  and listed in the {{format-registry}};
```

Comment:
I can see the registry growing to have hundreds of entries. I don't think suggesting that implementations should implement all of them is realistic.

Reply:
We need a middle ground between SHOULD and MAY for the registry.  Kind of a "hey it's a good idea if you want to, but you don't, no big deal."

Reply:
"Implementations are encouraged to X" is a pattern I've noticed in a few RFCs, is that the type of thing you're looking for?

e.g.
```
- SHOULD provide validation for each format attribute defined in this document;
- are encouraged to provide validation for format attributes listed in the {{format-registry}};
```

Reply:
I like "encouraged". How about something like, "implementations are encouraged implement formats from the registry and SHOULD (MUST?) provide an extension mechanism for formats they don't support".

The idea is that you either need to provide support for every format in the registry or provide a mechanism for the user to plug in support for anything that's missing.

Reply:
Updated